datastructures/Board.cpp

```diff
@@ -156,7 +156,7 @@ void Board::printUnicodeBoard() {
             if(i == 0)
                 std::cout << (char)('A' + j - 1) << "   ";
             else {
-                char32_t token = ' ';
+                std::string token = " ";
```

Comment:
can this be a char?

Reply:
It can't be a char sadly, doesn't fit. Unicode chars vary in size.